test/oauth-authorization-test.js

```diff
@@ -335,24 +351,38 @@ suite.addBatch({
                         requestToken(cl, this);
                     },
                     function(err, rt) {
+                        var self = this;
+
                         if (err) throw err;
-                        br = new Browser({runScripts: false});
-                        br.visit("http://localhost:4815/oauth/authorize?oauth_token=" + rt.token, this);
+                        br = new Browser({runScripts: false}); // faster without scripts...
+                        Step(
+                            function() {
+                                br.visit("http://localhost:4815/oauth/authorize?oauth_token=" + rt.token, this);
+                            },
+                            function() {
+                                self(!br.success, br);
+                            }
+                        );
                     },
                     function(err, br) {
+                        var self = this;
```

Comment:
@ShijuDekhDekh1, I think you have to do that yourself. If you are logged in in github, clicked the link in the email (view it on github) to the origin of the notification. Then you should be on the github page where the notification came from. On the right there should be a button Unsubscribe.